js/lunr-feed.js

```diff
@@ -0,0 +1,85 @@
+$(document).ready(function() {
+
+  function displaySearchResults(results, query) {
+    $('#results').empty().hide();
+    if (results.length) { // Are there any results?
+      var appendString = '<ul class="list--unstyled">'
+
+      $.getJSON("search_data.json", function(documents){
```

Comment:
Path handling needs to be improved here.

Reply:
For this and the other instance below, you can probably just point at an absolute path (i.e. `/search_data.json`) rather than the relative path here.